examples/universal/z_bytes.cxx

```diff
@@ -0,0 +1,135 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+#include <iostream>
+
+#include "stdio.h"
+#include "zenoh.hxx"
+using namespace zenoh;
+
+int _main(int argc, char** argv) {
+    // Numeric: u8, u16, u32, u128, usize, i8, i16, i32, i128, isize, f32, f64
+    {
+        const uint32_t input = 1234;
+        const auto payload = Bytes::serialize(input);
+        const auto output = payload.deserialize<uint32_t>();
+        assert(input == output);
+        // Corresponding encoding to be used in operations like `.put()`, `.reply()`, etc.
+        const auto encoding = Encoding("zenoh/uint32");
```

Comment:
Do we have `Encoding` constants instead of using the string form?

Reply:
not, yet will do in separate pr, since they are introduced in pico by https://github.com/eclipse-zenoh/zenoh-pico/pull/673